test/Base_Tests/src/System/File_Spec.enso

```diff
@@ -426,6 +426,13 @@ add_specs suite_builder =
         group_builder.specify "will resolve empty path to the current working directory" <|
             File.new "" . should_equal File.current_directory
 
+        group_builder.specify "try enso:// URL" <|
```

Comment:
- @otulach, at the end it was much more trivial problem
- no need for _integration tests_ (that starts the GUI)
- simple Enso unit test is enough